src/views/DefineCompany.vue

```diff
@@ -29,54 +29,89 @@
         </div>
       </v-card>
     </section>
+    <section class="mt-4" v-if="isEntityType">
+      <header>
+        <h2 v-if="entityFilter(EntityTypes.BCOMP)">2. Registered and Records Office Addresses</h2>
+        <h2 v-else-if="entityFilter(EntityTypes.COOP)">2. Registered Office Addresses</h2>
```

Comment:
As far as i know ( assumption 😆 ), most incorporations will have both addresses, Coops seem to be the exception.
Would it make sense to cut this down 1 line with 1 filter?  something like ...
`<h2>2. Registered <span v-if="entityFilter(EntityTypes.BCOMP)">and Records</span> Office Addresses</h2>`

Reply:
I prefer cleaner code (2 simple lines), but I'm flexible.

Reply:
Thats a great idea. Dint think that way. Will change the code

Reply:
Totally, all does the same thing, i'm fine with anything as well! 100% your call.